.github/workflows/CI.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+on:
+push:
+branches:[ develop ]
+jobs:
+ runs-on: ubuntu-latest:
```

Comment:
I don't think this will work. YAML works on indentation and it's wrong here.

If this works, you should see this (either passing or failing) on this PR:
![image](https://user-images.githubusercontent.com/56217868/123754547-a3f1cb00-d8d8-11eb-8f62-00eae899e9ab.png)


Please refer this: https://github.com/Real-Dev-Squad/website-backend/blob/develop/.github/workflows/test.yml